packages/legend-graph/src/graphManager/protocol/pure/v1/model/packageableElements/store/flatData/model/V1_FlatDataDataType.ts

```diff
@@ -68,13 +68,13 @@ export class V1_FlatDataDecimal extends V1_FlatDataNumber implements Hashable {
 }
 
 export class V1_FlatDataDate extends V1_FlatDataDataType implements Hashable {
-  dateFormat?: string | undefined;
+  dateFormat?: string[] | undefined;
   timeZone?: string | undefined;
 
   get hashCode(): string {
     return hashArray([
       CORE_HASH_STRUCTURE.FLAT_DATA_DATE,
-      this.dateFormat ?? '',
+      this.dateFormat === undefined ? '' : this.dateFormat.join(','),
```

Comment:
I think feel free to use `hashArray` here

Reply:
fixed